assets/activation-scripts/activate

```diff
@@ -108,10 +121,7 @@ fi
 #       container invocations, and it would have the incorrect value for
 #       nested flox activations.
 _FLOX_ENV="$($_coreutils/bin/dirname -- "${BASH_SOURCE[0]}")"
-if [ -n "${FLOX_ENV:-}" ] && [ "$FLOX_ENV" != "$_FLOX_ENV" ]; then
-  echo "WARN: detected change in FLOX_ENV: $FLOX_ENV -> $_FLOX_ENV" >&2
-fi
-export FLOX_ENV="$_FLOX_ENV"
+export FLOX_ENV="${FLOX_ENV:-$_FLOX_ENV}"
```

Comment:
nit
```suggestion
export FLOX_ENV="${FLOX_ENV:-$($_coreutils/bin/dirname -- "${BASH_SOURCE[0]}")}"
```
and drop `_FLOX_ENV`

Reply:
It's useful when debugging to see the explicit assignment of `_FLOX_ENV`. Without that you'll just see `FLOX_ENV=some path` and not know if that was inherited or assigned at runtime.